client/src/controllers/ActionController.ts

```diff
@@ -46,6 +46,12 @@ export class ActionController extends Controller<
   declare continueValue: boolean;
   declare urlValue: string;
 
+  select() {
+    if (this.element instanceof HTMLTextAreaElement) {
```

Comment:
We should also try to support the same method on InputElements. 

https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/select

Maybe instead, just return early if it's a button?

```
if (this.element instanceof HTMLButtonElement) return;
this.element.select();
```

That might work?